File: frontend/src/class/FeedWrite.ts
export default class FeedWrite {
  content: string; // 내용
  filePath: []; // 이미지, 동영상
  hashtags: []; // 해시태그

  constructor() {
    this.content = "";
    this.filePath = [];
    this.hashtags = [];
  }
}

File: Backend/src/main/java/com/ssafy/sunin/payload/request/feed/FeedWrite.java
package com.ssafy.sunin.payload.request.feed;

import lombok.*;
import org.springframework.web.multipart.MultipartFile;

import javax.validation.constraints.NotNull;
import java.util.List;

@Data
@AllArgsConstructor
public class FeedWrite {
    @NotNull
    private Long userId;
    private String content;
    private List<String> hashtags;
    private List<MultipartFile> files;
}